src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -231,7 +231,9 @@ export class MdAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
   }
 
   _handleKeydown(event: KeyboardEvent): void {
-    if (this.activeOption && event.keyCode === ENTER) {
+    if (event.keyCode === ESCAPE) {
+      this.panelOpen ? this.closePanel() : this._element.nativeElement.blur();
```

Comment:
Where does this `blur` put the focus? If it dumps it back to the `body`, then we don't want this behavior